### PR TITLE
Fix Scenario for CLEAR

### PR DESCRIPTION
**Pull request by wistuba**
- Fixes the problem we observed with CLEAR (the problem was that the scenario always return the same data chunk)
- Robust loading of the CLEAR dataset and fixing minor inaccuracies.
- Refactoring such that all time-incremental datasets (CLEAR, wild-time-data) use the same scenario

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: doc/benchmarking/renate_benchmarks.rst [hunk 1]
```diff
@@ -184,11 +184,8 @@ The first part contains all instances with classes 1 and 2, the second with clas
     * - Scenario Name
       - Description
       - Settings
-    * - :py:class:`~renate.benchmark.scenarios.BenchmarkScenario`
-      - Used in combination only with CLEAR-10 or CLEAR-100.
-      - * :code:`num_tasks`: Number of data partitions.
-    * - :py:class:`~renate.benchmark.scenarios.WildTimeScenario`
-      - Used in combination only with Wild-Time datasets. This is not the scenario used in the paper.
+    * - :py:class:`~renate.benchmark.scenarios.TimeIncrementalScenario`
+      - Used in combination only with Wild-Time datasets or CLEAR. This is not the scenario used in the paper.
```
Comment (610v4nn1) [inline doc/benchmarking/renate_benchmarks.rst:188]: nit: in which paper? both?

File: src/renate/benchmark/experiment_config.py [hunk 2]
```diff
@@ -331,7 +333,9 @@ def test_transform(dataset_name: str) -> Optional[Callable]:
     if dataset_name in ["CLEAR10", "CLEAR100"]:
         return transforms.Compose(
             [
-                transforms.Resize(224),
+                transforms.Resize(
```
Comment (610v4nn1) [inline src/renate/benchmark/experiment_config.py:336]: is this following the pre-processing described in the CLEAR paper?

**Review by 610v4nn1 — APPROVED** [review]

File: src/renate/benchmark/experiment_config.py (hunk 2 above)
Comment (wistuba) [reply src/renate/benchmark/experiment_config.py:336]: In the paper, they work on PIL images. Then this is not required. We work on Tensors. According to the documentation, this should bring it closer to what you would do when working on PIL images. Check the part on `antialiasing`: https://pytorch.org/vision/main/generated/torchvision.transforms.Resize.html

File: doc/benchmarking/renate_benchmarks.rst (hunk 1 above)
Comment (wistuba) [reply doc/benchmarking/renate_benchmarks.rst]: updated it

**Review by 610v4nn1 — APPROVED** [review]

**Pull request merged**